database/src/main/resources/db/migration/V1.95.0__RHCLOUD-29443_remove_secrets_columns.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE
+    "camel_properties"
+DROP COLUMN
+    "bearer_authentication";
+
```

Comment:
I don't think the column exists in `camel_properties`.
```suggestion
```

Reply:
You are right, yes... Thank you!